Documentation/toc.yml

```diff
@@ -140,6 +140,8 @@
     items:
     - name: Build Window
       href: Utilities/BuildWindow.md
+    - name: Build Window
+      href: DependencyWindow.md
```

Comment:
I'm not too familiar with the "toc" file. Does this line need to be in the utilities namespace?
  
`href: Utilities/DependencyWindow.md`

Reply:
Checking..